install_files/ansible-base/roles/ossec_server/tasks/authd.yml

```diff
@@ -1,7 +1,8 @@
 ---
-  # The authd ossec process automatically adds the app server ossec agent it is
-  # not an authenticated process. There is an option we could add to create and
-  # verify the server/agents ssl keys with the root CA cert. see
+  # The authd ossec process automatically adds the app server ossec agent.
+  # This addition is not an authenticated process. There is an option
+  # we could add to create and  verify the server/agents ssl keys with
+  # the root CA cert, see:
   # http://ossec-docs.readthedocs.org/en/latest/programs/ossec-authd.html
```

Comment:
This comment was related to #818 since we closed that issue we can remove this comment now too.